src/main/java/seedu/address/model/person/Address.java

```diff
@@ -53,6 +53,13 @@ public String getDisplayValue() {
         return value == EMPTY_ADDRESS ? PersonProfile.EMPTY_DISPLAY_VALUE : value;
     }
 
+    /**
+     * Returns true if value is empty.
+     */
+    public boolean isEmpty() {
+        return value.equals(EMPTY_ADDRESS);
```

Comment:
I think you could do value.isEmpty() which is a String library function that returns true if the String is an empty string, therefore you can get rid of the EMPTY_ADDRESS and other similar variables